packages/@chakra-icons/octicons/package.json

```diff
@@ -0,0 +1,41 @@
+{
+  "name": "@chakra-icons/octicons",
+  "version": "0.0.0",
+  "homepage": "https://github.com/kodingdotninja/chakra-icons/tree/main/packages/@chakra-icons/octicons",
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/kodingdotninja/chakra-icons"
+  },
+  "keywords": [
+    "chakra-ui",
+    "react",
+    "react-icons",
+    "chakra-icons",
+    "octicons"
+  ],
+  "main": "./dist/index.js",
+  "module": "./dist/index.mjs",
+  "types": "./dist/index.d.ts",
+  "files": [
+    "dist/",
+    "src/"
+  ],
+  "scripts": {
+    "build": "chakra-icons build -r 'https://github.com/primer/octicons' -n octicons -i icons -S snapshot.json -E",
```

Comment:
FYI, the url can be short with `primer/octicons`. but no problem and don't need to changes.

Reply:
I see, and it will automically added the github url prefix? cool

Reply:
That's the feature of https://github.com/Rich-Harris/degit